extra_tests/snippets/name.py

```diff
@@ -7,3 +7,7 @@
 import_func()
 
 assert __name__ == "__main__"
+
+assert __builtins__.__name__ == "builtins"
+import builtins
+assert builtins.__name__ == "builtins"
```

Comment:
The test code is good, but I want to suggest to replace it to `builtins_module.py` instead of here.